### PR TITLE
Add Welcome

### PR DESCRIPTION
**Pull request by CoolPlayLin**

### CODE REVIEW

**Comment by acheong08** [comment]
failed again lol

**Comment by acheong08** [comment]
I know the issue this time. Will fix

**Review by acheong08 — APPROVED** [review]

File: .github/workflows/pull-requests.yml [hunk 1]
```diff
@@ -67,11 +65,21 @@ jobs:
...
+              Your PR will review by ${reviewer}
+              We look forward to reviewing your Pull request shortly ✨
+              `
+            })
       - name: Assign to dev branch
-        if: always()
+        if: github.ref != 'refs/heads/dev'
         uses: actions/github-script@v6.4.1
         with:
-          github-token: ${{ secrets.GH_TOKEN }}
```
Comment (acheong08) [inline .github/workflows/pull-requests.yml:74]: will it work without github token?

Comment (CoolPlayLin) [reply .github/workflows/pull-requests.yml:74]: 没有的话默认是GITHUB_TOKEN

**Pull request merged**